layers/+lang/common-lisp/config.el

```diff
@@ -10,3 +10,8 @@
 ;;; License: GPLv3
 
 (spacemacs|define-jump-handlers lisp-mode slime-inspect-definition)
+
+;; Company integration
+(spacemacs|defvar-company-backends lisp-mode)
+(defcustom enable-slime-company t
```

Comment:
We normally use `defvar` for layer variables, not `defcustom`. The name might need a `lisp-` or `spacemacs-` prefix as well, but I'm not sure we have a defined convention for the prefix of layer variables.